basic/src/test/java/org/jboss/hal/testsuite/test/configuration/undertow/applicationsecuritydomain/ApplicationSecurityDomainOperations.java

```diff
@@ -0,0 +1,110 @@
+package org.jboss.hal.testsuite.test.configuration.undertow.applicationsecuritydomain;
+
+import org.apache.commons.lang3.RandomStringUtils;
+import org.jboss.dmr.ModelNode;
+import org.jboss.hal.testsuite.dmr.ModelNodeGenerator;
+import org.jboss.hal.testsuite.test.configuration.elytron.ElytronOperations;
+import org.jboss.hal.testsuite.util.ConfigUtils;
+import org.wildfly.extras.creaper.core.online.OnlineManagementClient;
+import org.wildfly.extras.creaper.core.online.operations.Address;
+import org.wildfly.extras.creaper.core.online.operations.Operations;
+import org.wildfly.extras.creaper.core.online.operations.Values;
+
+import java.io.IOException;
+
+class ApplicationSecurityDomainOperations {
```

Comment:
It seems that this class is completely replaced by `UndertowElytronOperations`. If this is true, please remove `ApplicationSecurityDomainOperations`.